### DISCUSSION TITLE
A single product in multiple groups/categories

### DISCUSSION THREAD

**Discussion by alinjie** [saleor/saleor#8403] [Q&A] [answered]
Hi and thank you for an amazing project! I have a question with regards to how to organize products within Saleor.

**Consider having the following categories:**
Men's Shoes
Women's Shoes

**And the following product:**
NB Running Shoes

In this case the product should belong to both categories. How would I go about grouping them since adding a product to multiple categories is not possible? Should I consider naming the Category "Shoes" and grouping by collections instead? Any tips would be appreciated, as I'm pretty new to Saleor.

**Comment by karolkielecki** [comment, marked as answer]
@alinjie Hi! Thanks for your kind words! Yes, you can assign one product to only one category in the dashboard. 

Maybe you can simply create two products, e.g., NB Running Shoes (Men) and NB Running Shoes (Women), and then assign them to these two categories. Or you can create a category called "unisex shoes".

**Reply by alinjie** [reply]
Hello and thanks for your reply!

I guess I could create two different products, but that seems a little less than ideal since I would now have to manage the two products separately and I could in theory risk having different prices for the two.

I guess the latter solution is more viable in this case, even though being able to place a product in multiple categories would be the prefect solution. Any idea if this will be possible in the future?

**Reply by karolkielecki** [reply]
@alinjie Yes, for now, the 2nd solution would be better in your case. I'll discuss it with our team and will let you know as soon as possible.

**Reply by alinjie** [reply]
@karolkielecki Thank you! I will leave the discussion open in the meantime 😊

**Reply by karolkielecki** [reply]
@alinjie Currently, it's not on our roadmap. We have other priorities, which you can find here: https://github.com/saleor/saleor/projects/12. If there are any changes regarding this case, I'll let you know. So, for now, I would go with the 2nd solution mentioned above. Cheers!

**Reply by alinjie** [reply]
@karolkielecki Thanks for taking the time to investigate. It would be nice to see this feature in the future. Have great weekend!
